mysql-test/include/commit.inc

```diff
@@ -620,14 +620,14 @@ create temporary table t2 (a int);
 # One commit for the create temporary table, and two for committing the
 # read of the stored procedure from Aria table (creating temporary table
 # clears the sp cache).
-call p_verify_status_increment(3, 0, 2, 0);
+call p_verify_status_increment(5, 0, 2, 0);
```

Comment:
the comment above this accounts for 3 of the increments. Explain the other two if keeping this functionality.